app/javascript/react/components/RestaurantTile.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Link } from "react-router-dom"
+
+const RestaurantTile = (props) => {
+
+  return (
+    <div>
+      <h1>
+        <Link to={`restaurant/${props.id}`}>
+          {props.name}
+        </Link > 
+      </h1>
+    </div>
```

Comment:
`<div>`'s aren't _super_ necessary here since the `<h1>` could be your top level element